modules/nf-core/mtmalign/align/main.nf

```diff
@@ -0,0 +1,46 @@
+
+
+process MTMALIGN_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::mtm-align=20220104"
```

Comment:
Should we also update it to this? 

https://github.com/nf-core/modules/blob/0f4f5205a67d3a90adc8e341cf2b19a16f5cb272/modules/nf-core/muscle5/super5/main.nf#L4

Reply:
true, we should – I'm not sure its documented anywhere, but it is apparently now the preferred way to have conda envs.